stratum/hal/lib/barefoot/bfrt_table_manager_test.cc

```diff
@@ -578,11 +597,6 @@ TEST_F(BfrtTableManagerTest, DeleteDigestEntrySuccess) {
 
   const std::string kDigestEntryText = R"pb(
     digest_id: 401732455
-    config {
```

Comment:
shall we maintain this test and verify that brft_table_manager does not care? It seems to me another test we get for free.